src/blockchain/lending/helper.ts

```diff
@@ -0,0 +1,216 @@
+import { BaseProvider } from '@ethersproject/providers'
+import {
+  LENDING_COMPTROLLER,
+  LENDING_LIQUIDITY_POOL,
+  LENDING_LPT,
+  LENDING_X_AAVE_A_MARKET,
+  LENDING_X_AAVE_A_PRICE,
+  LENDING_X_AAVE_B_MARKET,
+  LENDING_X_AAVE_B_PRICE,
+  LENDING_X_INCH_A_MARKET,
+  LENDING_X_INCH_A_PRICE,
+  LENDING_X_INCH_B_MARKET,
+  LENDING_X_INCH_B_PRICE,
+  LENDING_X_KNC_A_MARKET,
+  LENDING_X_KNC_A_PRICE,
+  LENDING_X_KNC_B_MARKET,
+  LENDING_X_KNC_B_PRICE,
+} from '@xtoken/abis'
+
+import {
+  Comptroller,
+  LiquidityPool,
+  LPT,
+  Market,
+  XAAVEPrice,
+  XINCHPrice,
+  XKNCPrice,
+} from '../../types'
+import { getContract } from '../utils'
+
+export const getComptroller = async (provider: BaseProvider) => {
+  const network = await provider.getNetwork()
+  const comptroller = getContract(
+    LENDING_COMPTROLLER,
+    provider,
+    network
+  ) as Comptroller
+  if (!comptroller) {
+    return Promise.reject(new Error('Could not create Comptroller Contract'))
```

Comment:
Any suggestions on how we can standardise this error message? Was using `Unknown error` so far at other places where we're unable to initialise the contracts. But would definitely prefer to have an `Errors` object created within `constants.ts` and use it across which can be expanded for other error messages too, something like below:
```
const Errors = Object.freeze({
  CONTRACT_NOT_FOUND: 'Failed initialising contract',
})
```